.github/workflows/ci_tests.yml

```diff
@@ -220,16 +189,15 @@ jobs:
       ENABLE_TRILINOS: OFF
       GCP_BUCKET: geosx/integratedTests
       RUNS_ON: ubuntu-22.04
-
+        
   # If the 'ci: ready to be merged' PR label is found, the cuda jobs run immediately along side linux jobs.
   # Note: CUDA jobs should only be run if PR is ready to merge.
```

Comment:
Same comment and note should be added for the `run_integrated_tests` job.

Reply:
done